pkg/policy/api/rule.go

```diff
@@ -66,22 +66,23 @@ type Rule struct {
 type Entity string
 
 const (
-	// World is an entity that represents traffic external to endpoint environment
-	World Entity = "world"
-	// Host is an entity that represents traffic within endpoint host
-	Host Entity = "host"
+	// EntityWorld is an entity that represents traffic external to endpoint environment
+	EntityWorld Entity = "world"
+	// EntityHost is an entity that represents traffic within endpoint host
```

Comment:
within the endpoint's cluster

Reply:
Host entity represents whole cluster?

Reply:
Let's be specific:

EntityHost is the entity that represents the host on which the endpoint is running on.